build.gradle

```diff
@@ -16,6 +16,9 @@ buildscript {
         // Workaround for Lint + Retrolambda.
         classpath 'me.tatarka.retrolambda.projectlombok:lombok.ast:0.2.3.a2'
 
+        // Generates build info without breaking incremental builds
+        classpath 'hu.supercluster:paperwork-plugin:1.2.1'
```

Comment:
It would be really nice if I could reference it like ```classpath libraries.paperworkPlugin```, because this way the version code is written twice, and I'd like it to be DRY. But that would need the libraries block to come before this block, else gradle will not pick it up.

Or, even better, I'd suggest to extract that block into a standalone gradle file altogether, and include it at the beginning of this script.